src/string.cr

```diff
@@ -3991,6 +3995,10 @@ class String
     true
   end
 
+  def ends_with?(re : Regex)
+    !!($~ = /#{re}\z/.match(self))
```

Comment:
Doesn't this require `/(?:#{re})\z/`?

Reply:
Needless, `re.to_s` yields enclosed string.

Reply:
Oh, cool.